### ISSUE TITLE
Newest release has error. See attached image. 

### ISSUE THREAD

**Issue by dnoneill** [recogito/annotorious-openseadragon#128]
When trying to load the newest release the following error occurs.

<img width="1200" alt="Screen Shot 2022-06-15 at 3 07 21 PM" src="https://user-images.githubusercontent.com/19173991/173905716-b1ff8d17-32b5-466b-aaee-91f14a03f07c.png">

**Comment by darwinjob** [comment]
I confirm this. First of all, if you click on the release link you will see something weird:
![image](https://user-images.githubusercontent.com/358762/174487533-4cdb61e2-5755-4cb2-b8ce-d04f87d3050c.png)

This is broken:
```
<script src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/3.1.0/openseadragon.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/@recogito/annotorious-openseadragon@2.7.5/dist/openseadragon-annotorious.min.js"></script>
```

However, this works:
```
<script src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/3.1.0/openseadragon.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/@recogito/annotorious-openseadragon@2.7.6/dist/openseadragon-annotorious.min.js"></script>
```

**Comment by dnoneill** [comment]
@rsimon The library I used was the release in the screenshot that I downloaded and loaded. I use script tags and use the order that I used before which is openseadragon, annotorious, openseadragon-annotorious

**Comment by rsimon** [comment]
Yikes - sorry for this. Now I remember... ;-) Yes, I managed to build & release v2.7.5 with a broken build config. I did fix the bug, and pushed v2.7.6 to the npm registry (and the CDN). But then forgot to publish a ZIP package on GitHub. (Lesson learned: never do this stuff late at night...)

So as an immediate fix: you can pull the latest version - which is actuall v2.7.6 now - from the CDN here:

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@recogito/annotorious-openseadragon@2.7.6/dist/annotorious.min.css">

<!-- JS -->
<script src="https://cdn.jsdelivr.net/npm/@recogito/annotorious-openseadragon@2.7.6/dist/openseadragon-annotorious.min.js"></script>
```

I'll roll a GitHub release for v2.7.6, too, unpublish the broken v2.7.5, and close the issue when that's done.

Cheers, thanks for reporting & sorry for the inconvenience!
